cogboard-webapp/src/components/DropdownField.js

```diff
@@ -83,3 +102,4 @@ DropdownField.defaultProps = {
 };
 
 export default DropdownField;
+//margin-top: 16px;
```

Comment:
Unnecessary comment?

Reply:
Yes, forgot to remove it before committing. Gone now.